hydra-node/src/Hydra/Options.hs

```diff
@@ -340,7 +340,7 @@ cardanoVerificationKeyFileParser =
         <> metavar "FILE"
         <> help
           "Cardano verification key of another party in the Head. Can be \
-          \provided multiple times, once for each participant."
+          \provided multiple times, once for each participant (current maximum limit is 4)."
```

Comment:
You forgot to use `maximumNumberOfParties` here :)

Reply:
:man_facepalming: